script/c94207108.lua

```diff
@@ -121,7 +121,7 @@ end
 function s.sptg2(e,tp,eg,ep,ev,re,r,rp,chk)
 	if chk==0 then return Duel.GetLocationCount(tp,LOCATION_MZONE)>0
 		and Duel.IsExistingMatchingCard(s.spfilter2,tp,LOCATION_GRAVE,0,1,nil,e,tp) end
-	Duel.SetOperationInfo(0,CATEGORY_SPECIAL_SUMMON,nil,1,0,0)
+	Duel.SetOperationInfo(0,CATEGORY_SPECIAL_SUMMON,nil,1,tp,LOCATION_GRAVE)
```

Comment:
Marincess Wonder Heart: Declare that the Special Summon is from the GY
I'm not 100% sure if the `tp` is correct here, our version of the operation parameters are somewhat confusing after being changed and unchanged in pursuit of implementing obscure Pot of Duality vs Token rulings.